MapboxNavigation/CarPlayManager.swift

```diff
@@ -142,20 +153,24 @@ public class CarPlayManager: NSObject {
     /**
      The bar button that shows the selected route overview on the map.
      */
-    @objc public lazy var overviewButton: CPMapButton = {
-        let overviewButton = CPMapButton { button in
+    @objc public lazy var trackingStateButton: CPMapButton = {
```

Comment:
Maybe call it `userTrackingButton` by analogy with MapKit’s MKUserTrackingButton.

/ref mapbox/mapbox-gl-native#1093